.gitignore

```diff
@@ -141,8 +141,11 @@ out/
 /broadcast/*/1337/
 /broadcast/**/dry-run/
 
-# Docs
-docs/
+# drawio backups
+docs/images/.$*.bkp
```

Comment:
1. Does the draw.io backup need to be added to a gitignore? Or is this just something on your machine?
2. There is also an argument for it being kept in source control, otherwise changes might always rely on you having your local copy. If the backup was in source control, any team member could make changes as/when needed. wdyt?

Reply:
The program that I'm using (draw.io) sometimes makes one, but I'll just get rid of the file and this ignore.